forte.yml

```diff
@@ -301,8 +301,9 @@ Layer:
     <<: *db
     table: |-
         ( SELECT
-            geometry, '{lang}' as lang, type, capital, NULL as ldir,
+            geometry, '{lang}' as lang, type, NULL as ldir,
             CASE WHEN type='city' THEN 1 WHEN type='state' THEN 2 ELSE 100 END AS prio,
```

Comment:
Are we sure we cant the `place=state` to be ordered even after cities with `capital=10` ?

Reply:
Mmmh ok

https://github.com/tilery/pianoforte/pull/45/commits/14f4e0a473d82935829753fcc30bdd843f3d0b6d changes it like this:

* Capital = yes
* Capital IN 2,3,4 (regionals)
* State
* Other capitals

What do you think